docs/contribute/writing-articles.md

```diff
@@ -0,0 +1,400 @@
+
+### Description
+
+In order to have descriptive info for search engines and embeds, you have to add a description to the front matter. To do so, you need the following code at the start of your article:
+
+```md
+---
+description: Example description
+---
+```
+
+### Previous & Next
+
+You may have seen previous or next article buttons at the bottom of wiki pages. They can be enabled or disabled in the frontmatter section, using `prev` and `next`.
+
+By default, the previous and next buttons are disabled. To enable these buttons, you need to set the options to true:
+
+```md
+---
+prev: true // [!code ++]
+next: true // [!code ++]
+description: Example description
+---
+```
+
+::: danger IMPORTANT
+The frontmatter section **must** be at the beginning of the file, otherwise it will not work. For example, `correct.md` will display a correct markdown file, and `incorrect.md` will display an incorrectly written markdown file.
+
+:::code-group
+```md [correct.md]
+---
+prev: false
+next: false
+description: Example description
+---
+
+# Example Article
+```
+
+```md [incorrect.md]
+<!-- Extra space causes errors -->
+---
+prev: false
+next: false
+description: Example description
+---
+
+# Example Article
+```
+:::
+
+## Formatting
+
+### Markdown
+
+This wiki uses a modified version of markdown (.md) files for articles, allowing a fairly easy process to write a new article.
+Markdown essentially is a simple text file with "shortcuts" that help with formatting.
+An example of this is in Discord, where you can easily format your messages to be bold, italicized, etc.
+
+::: tip
+[This Markdown Guide](https://www.markdownguide.org/basic-syntax/) goes into greater detail than the basics given in this article.
+:::
+
+#### Common Usages
+
+There are a few simple ways to format text.
+
+- Using `**` on either side of a word/phrase will **bold that word/phrase**.
+- Using `_` on either side of a word/phrase will _italicize that word/phrase_.
+- Using `***` on either side of a word/phrase will ***bold and italicize that word/phrase***.
+- Using `~~` on either side of a word/phrase will ~~strike that word/phrase out~~.
+
+::: warning
+These symbols generally ***must not*** have spaces between them and the word(s) you are formatting.
+:::
+
+#### Headings
+
+You may want headings in your article. These are places you can link to which will bring readers to that exact spot on the page. For example, [this links to the formatting heading](#formatting).
+
+There are several different heading "sizes" you can use.
+
+- Using `#` will create the main heading (h1). For example, [this links to a main heading](#contributing-articles).
+- Using `##` will create the an h2 heading.
+
+This pattern continues until `######`, which creates an h6 heading.
+
+::: info
+Unless there is an empty line between lines of text, headings, or other line symbols, the lines of text, heading, code, or whatever else will either not work or be separated as intended.
+
+::: code-group
```

Comment:
this is not a change request this info box is just really cool

great work

Reply:
VitePress can do some cool stuff :)